frontend/packages/component-library/src/tooltip/_Tooltip.tsx

```diff
@@ -26,6 +26,7 @@ export interface TooltipProps extends HTMLAttributes<HTMLDivElement> {
   size?: ComponentSizeXSToL;
   /** Tooltip custom styles (used for positioning the tooltip on the go) */
   style?: React.CSSProperties;
+  hideTail?: boolean;
```

Comment:
super nit: can you please add a short comment for this prop for documentation (just like other props have comments)

Reply:
Updated!